### ISSUE TITLE
The repository countersignature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain

### ISSUE THREAD

**Issue by jsoques** [tryphotino/photino.Samples#8]
I get this error doing dotnet build on Blazor sample (most other samples also get this error):

> error NU3028: Package 'Microsoft.Extensions.Logging.Console 5.0.0' from source 'https://api.nuget.org/v3/index.json': The author primary signature's timestamp found a chain building issue: UntrustedRoot: self signed certificate in certificate chain

dotnet --info:

```
.NET SDK (reflecting any global.json):
 Version:   5.0.201
 Commit:    a09bd5c86c

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  20.10
 OS Platform: Linux
 RID:         ubuntu.20.10-x64
 Base Path:   /usr/share/dotnet/sdk/5.0.201/

Host (useful for support):
  Version: 5.0.4
  Commit:  f27d337295

.NET SDKs installed:
  3.1.407 [/usr/share/dotnet/sdk]
  5.0.201 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.13 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.4 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.13 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.4 [/usr/share/dotnet/shared/Microsoft.NETCore.App]


```

**Comment by MikeYeager** [comment]
That is quite strange. This is the only report we've seen of this. The problem appear to be with a Microsoft package "Microsoft.Extensions.Logging.Console 5.0.0", not one of our packages. Perhaps try re-installing the .NET 5 SDK? It may be how it was installed. We've see issues trying to install it from a package manager. Our Quick Start Guide mentions this under dependencies:

On Linux this cannot be installed through the Snap package manager and must be installed using the [scripted install method described here](https://docs.microsoft.com/en-us/dotnet/core/install/linux-scripted-manual#scripted-install).

**Comment by jsoques** [comment]
> That is quite strange. This is the only report we've seen of this. The problem appear to be with a Microsoft package "Microsoft.Extensions.Logging.Console 5.0.0", not one of our packages. Perhaps try re-installing the .NET 5 SDK? It may be how it was installed. We've see issues trying to install it from a package manager. Our Quick Start Guide mentions this under dependencies:
> 
> On Linux this cannot be installed through the Snap package manager and must be installed using the [scripted install method described here](https://docs.microsoft.com/en-us/dotnet/core/install/linux-scripted-manual#scripted-install).

Thanks, I will try reinstalling .Net 5 SDK.

**Comment by jsoques** [comment]
Cleaning my system of all traces of .NET Core and installing v5.0 did not work. After quite a struggle since some of my other projects depend on v3.1, I reinstalled SDK 3.1 and 5.0 but the above error still persisted when trying to `dotnet run` the Photino.Blazor example. Then I found this... https://github.com/NuGet/Home/issues/10491#issuecomment-778841003 that did the trick. So apparently some certs in nuget.org are expired or what not. I am now able to `dotnet run` the Photino.Blazor example although it gives an error (`File "app://app/" could not be found.`) that is mentioned in another issue I will look at.
